.github/workflows/test_with_drupal.yaml

```diff
@@ -0,0 +1,34 @@
+name: Test With Drupal
+
+on:
+    pull_request: null
+    push:
+        branches:
+            - master
+
+jobs:
+    test_with_drupal:
+        runs-on: ubuntu-latest
+        steps:
+            -   uses: actions/checkout@v2
+            -   uses: shivammathur/setup-php@v1
+                with:
+                    php-version: 7.3
+                    coverage: none
+
+            -   name: Install Drupal project
+                run: |
+                    # Install latest Drupal version
```

Comment:
`composer install` is needed, so Rector has it's own dependencies too

Reply:
@TomasVotruba 
I thought `composer require rector/rector-prefixed:0.6.13 --dev` is what would bring rector-prefixed with all its dependencies...

Locally I run rector-prefixed from `vendor/bin/rector`, but through github actions I got an error.
In the meantime I switched it back to `bin/rector` as it was in the original script.

Reply:
That installs remote version, which will be definitely different from local.